roles/logging/tasks/filebeat.yml

```diff
@@ -52,3 +56,7 @@
 
 - name: start and enable filebeat service
   service: name=filebeat state=started enabled=yes
+  register: result
```

Comment:
probably don't need a retry on the service start?  unless you've seen issues here ?